code/modules/clothing/under/syndicate.dm

```diff
@@ -272,3 +272,13 @@
 	. = ..()
 	var/obj/item/clothing/accessory/maidapron/inteq/A = new (src)
 	attach_accessory(A)
+
+/datum/outfit/syndicate/assult_komodo
```

Comment:
delet this outfit, it's in the wrong place and a copy of the one you actually want to be using